<?xml version="1.0" ?>
<sdf version="1.6">
  <model name="goal_marker">
    <static>true</static>
    <link name="link">
      
      <!-- First bar of the cross -->
      <visual name="cross_bar_1">
        <pose>0 0 0 0 0 0</pose>
        <geometry>
          <box>
            <size>0.5 0.05 0.05</size>
          </box>
        </geometry>
        <material>
          <ambient>1 0 0 0.6</ambient>
          <diffuse>1 0 0 0.6</diffuse>
        </material>
        <transparency>0.4</transparency>
      </visual>

      <!-- Second bar of the cross -->
      <visual name="cross_bar_2">
        <pose>0 0 0 0 0 1.5708</pose> <!-- 90Â° rotation around Z -->
        <geometry>
          <box>
            <size>0.5 0.05 0.05</size>
          </box>
        </geometry>
        <material>
          <ambient>1 0 0 0.6</ambient>
          <diffuse>1 0 0 0.6</diffuse>
        </material>
        <transparency>0.4</transparency>
      </visual>

    </link>
  </model>
</sdf>
